app/src/main/java/com/internship/move/data/model/User.kt

```diff
@@ -5,7 +5,13 @@ import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
 data class User(
+    @Json(name = "__v") val v: Int,
```

Comment:
See which fields you actually need to be parsed from Json for this class and remove the unnecessary ones. Holding only the necessary info will reduce your app's memory footprint.
Also, remember to periodically format your code